packages/react-components/react-combobox/library/src/components/Dropdown/useDropdown.tsx

```diff
@@ -59,6 +62,18 @@ export const useDropdown_unstable = (props: DropdownProps, ref: React.Ref<HTMLBu
     },
   });
 
+  const { targetDocument } = useFluent();
+
+  useOnClickOutside({
```

Comment:
Please add a cypress tests for this change. There are none currently, and it would be nice to start building them up because this control is getting more complex and still has no e2e testing.

No need to do test everything! just this behaviour

Reply:
Setting it up should be quite easy https://github.com/microsoft/fluentui/tree/master/tools/workspace-plugin/src/generators/cypress-component-configuration

Reply:
Added e2e tests to both Combobox and Dropdown components